app/routes.py

```diff
@@ -17,6 +17,8 @@
 from models.forms import (AdminForm, LoginForm, RegistrationForm, UploadForm,
                           UserCriteria, UserProfileForm)
 
+from app import bcrypt
+
 class User(UserMixin):
     pass

```

Comment:
```suggestion
```

Reply:
AFAIK it is recommended to put `user_loader` in the same module where a user model is defined